ext/opcache/Optimizer/pass3.c

```diff
@@ -113,47 +113,13 @@ void zend_optimizer_pass3(zend_op_array *op_array, zend_optimizer_ctx *ctx)
 						}
 					}
 
-					if ((opline->op1_type & (IS_VAR | IS_CV))
+					if (opline->opcode >= ZEND_ADD
+					    && opline->opcode <= ZEND_POW
```

Comment:
We may want to define ZEND_FIRST_BINOP and ZEND_LAST_BINOP to avoid breaking this kind of code when new operators are added.